charts/datadog/values.yaml

```diff
@@ -605,19 +605,16 @@ datadog:
   ## Enable process agent and provide custom configs
   processAgent:
     # datadog.processAgent.enabled -- Set this to true to enable live process monitoring agent
-
-    ## Note: /etc/passwd is automatically mounted when `processCollection` or `processDiscovery` is enabled.
+    # DEPRECATED. Set `processCollection` or `containerCollection` instead.
```

Comment:
```suggestion
    # DEPRECATED. Set `datadog.processAgent.processCollection` or `datadog.processAgent.containerCollection` instead.
```

Reply:
do you anticipate this deprecation could break any existing setup for users?

Reply:
No I don't believe so, it was only used in the two instances I replaced. The `containerCollection` config essentially supersedes its functionality in terms of enabling the process agent container by default but also allows customers to turn it off in a more obvious way.